boards/common/particle-mesh/include/board.h

```diff
@@ -95,6 +95,11 @@ extern "C" {
 #define LED2_MASK           (1 << 15)
 #define LED_MASK            (LED0_MASK | LED1_MASK | LED2_MASK)
 
+/* Both are typically the case when saul_default is on, but strictly, it is
+ * those two that in combination make LEDs effectively unavailable to users.
+ * Not defining LEDx_ON has the effect of making LEDx_IS_PRESENT undefined. */
+#if !(IS_USED(MODULE_AUTO_INIT_SAUL) && IS_USED(MODULE_SAUL_PWM))
```

Comment:
Maybe add a comment here as to why the LEDs are only available on those boards iff `saul_pwm` is unused (basically the info from the PR description).

Reply:
Extended and slightly reworded in a fix, thanks for pointing this out.

Reply:
Thanks!